src/documentation/pages/components/pagination/examples/_additional-sizes.vue

```diff
@@ -0,0 +1,245 @@
+<template lang="pug">
+  <ComponentExample title="Additional Sizes" id="additional_sizes" :tabs="exampleTabs" padding="-p--0">
+    p.-text(slot="example-description")
+      | Pagination supports the following sizes: <code>xs</code>, <code>sm</code>, <code>md</code> (default), <code>lg</code>, and <code>xl</code>.
+    <Wrapper slot="example">
```

Comment:
no need for wrapper here, slot="example" could go in the below div as it's 1 child